config.yml.example

```diff
@@ -92,6 +92,8 @@ measurement:
     #--- MacOS: On Mac you only need this provider. Please delete all others!
       powermetrics.provider.PowermetricsProvider:
         resolution: 100
+      cpu.utilization.macos.system.provider.CpuUtilizationMacosSystemProvider:
```

Comment:
This is new convention to add the os here. Maybe we should move to the same syntax/ directory strucutre as we have in config with the os being the first?